db/seeds.rb

```diff
@@ -1,7 +1,37 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'nokogiri'
+
+module JokeParser
+  def self.setup
+    seed_laffy_taffy
+    seed_chartcons
+  end
+
+  def self.seed_laffy_taffy
+    doc = Nokogiri::HTML(File.open('db/sources/laffy_taffy.html'))
+
+    doc.search('ol > li').map do |element|
+      joke = element.inner_text.strip.gsub(/^\d+\W\s/, "").downcase.capitalize.split("?")
+      question = "#{joke.first}?"
+      answer = joke.last.strip.gsub(/^\W */, "").capitalize
```

Comment:
it's also stripping out the dash that's in some answers as well so we'll need to strip that out 👍